guide/english/python/files-and-io/index.md

```diff
@@ -106,6 +106,16 @@ Sample code to open a text file using the `with` statement:
 with open('hello_world.txt', 'w') as f:
     f.write('Hello World!')
 ```
+#### Reading files
+There are many ways of reading a file. For a file <code>fh</code> opened in the read mode, the following are the options:
+
+<code>fh.read()</code>: read file as a single string
+
+<code>fh.read(n)</code>: read n number of characters
+
+<code>fh.readline()</code>: read one line and end with <code>'\n'</code>
```

Comment:
there is no need to add the exclamation marks around `\n` because the `<code>` are already providing emphasis